corehq/apps/smsbillables/management/commands/bootstrap_yo_gateway.py

```diff
@@ -0,0 +1,47 @@
+from decimal import Decimal
+import logging
+
+from django.core.management.base import LabelCommand
+
+from corehq.apps.accounting.models import Currency
+from corehq.apps.sms.models import INCOMING, OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+
+
+logger = logging.getLogger('accounting')
+
+
+def bootstrap_yo_gateway(orm):
+    ugx, _ = (orm['accounting.Currency'] if orm else Currency).objects.get_or_create(code='UGX')
+    sms_gateway_fee_class = orm['smsbillables.SmsGatewayFee'] if orm else SmsGatewayFee
+    sms_gateway_fee_criteria_class = orm['smsbillables.SmsGatewayFeeCriteria'] if orm else SmsGatewayFeeCriteria
+
+    SmsGatewayFee.create_new(
+        'YO',
+        INCOMING,
+        Decimal('55.0'),
+        currency=ugx,
+        fee_class=sms_gateway_fee_class,
+        criteria_class=sms_gateway_fee_criteria_class,
+    )
+
+    SmsGatewayFee.create_new(
+        'YO',
```

Comment:
Yeah it looks like we have the right id passed as backend_instance already right? (we only want HTTP + backend id for outbound, the inbound one is correct)